modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -221,7 +221,7 @@ High value can indicate nodes failing and causing leadership changes.
 
 Redpanda process utilization.
 
-Shows the true utilization of the CPU by a Redpanda process.
+Shows the true utilization of the CPU by a Redpanda process. Check this metric on a per-shard basis. If a shard is at 100% utilization for a continuous period of time, it likely results in high latency.
```

Comment:
@ztlpn Is this guidance OK to provide to users? Could we provide a bit more information such as how we should define "continuous period of time", and is there a specific way to calculate per shard? Our doc says
```
sum(vectorized_reactor_utilization{instance=~"[[node]]",shard=~"[[node_shard]]"}) by ([[aggr_criteria]])
```
But I am not very familiar with the syntax inside the {} brackets. Thanks!

Reply:
> Is this guidance OK to provide to users?

Yes, looks OK. We can also mention that high latency will be observed only for partitions assigned to that core and mention the need for balancing.

> how we should define "continuous period of time"

I would say more than a few seconds, but it really depends on the workload. The user may not care for small latency spikes here and there if they do some kind of bulk loading. OTOH they would want it to never reach 100% if they are doing realtime processing.

> is there a specific way to calculate per shard

Inside `{}` are filters - if you want to view utilization for specific nodes and/or shards you specify them there, but usually I would start with viewing all cores of the cluster. As the metric has per-shard granularity and we want to view it per-shard, we don't really need `sum` either. So we can just mention the metric name and that it has per-shard granularity and let the user figure out the way they want to filter it themselves.